test/models/category_test.rb

```diff
@@ -1,5 +1,22 @@
 require 'test_helper'
 
 class CategoryTest < ActiveSupport::TestCase
+  test "it has attributes" do
+    category = build(:category)
+    assert category.valid?
+  end
 
+  test "category is not valid without a name" do
+    category = build(:category, name: nil)
+    category = build(:category, name: "  ")
```

Comment:
there should be two different category variable names here if we're testing both.